Client/Ecosia/UI/NTP/CircleButton.swift

```diff
@@ -3,19 +3,30 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import UIKit
+import Shared
 
 class CircleButton: ToolbarButton {
-    struct Config {
-        let hideCircle: Bool
-        let image: String
-        let margin: CGFloat
-
-        static var search: Config {
-            return .init(hideCircle: false, image: "search", margin: 8)
+    enum Config {
+        case search
+        case newTab
+        
+        var image: String {
+            switch self {
+            case .search: return "search"
+            case .newTab: return "nav-add"
+            }
         }
-
-        static var newTab: Config {
-            return .init(hideCircle: true, image: "nav-add", margin: 8)
+        var hideCircle: Bool {
```

Comment:
- can this `hideCircle` be made `private` ?
- it seems to be a get only computed `var` rather than an imperative function that results into the command of hiding the circle. Would it be possible to make it `isCircleHidden` ?

Reply:
Nice suggestion on changing the name, I think I will use `shouldHideCircle` since it is basically telling us whether it should be hidden, not the other way 🙂 let me know what you think.

On the second point, now that it is part of the `enum`, it can't be made `private` since it is used on the class.